pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -337,3 +349,44 @@ class DualRK45PseudoStepper(DualRKVdH2RPseudoStepper):
     @property
     def _pseudo_stepper_order(self):
         return 4
+
+
+class DualDenseRKPseudoStepper(BaseDualPseudoStepper):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        scheme = resource_string(
+           'pyfr.integrators', '{0}/{1}'.format('schemes', self.path)
```

Comment:
format with hardcode string.

Reply:
What do you mean? 
resource_string('pyfr.integrators.schemes', self.path)
doesn't work unless we place __init__.py within /schemes, right? This can mess up the regex.

Reply:
'/schemes/' + self.path.  The first argument to format is hardcoded.